src/init.cpp

```diff
@@ -1508,7 +1508,7 @@ bool AppInitMain()
             }
 
             fLoaded = true;
-            LogPrintf(" block index %15dms\n", GetTimeMillis() - load_block_index_start_time);
+            LogPrint(BCLog::BENCH, "- Loaded block index: %dms\n", GetTimeMillis() - load_block_index_start_time);
```

Comment:
@romanz I didn't meant to change the log category. I think some of these messages must/should be present in the log regardless of the enabled logging categories.

Reply:
Updated the PR to use the original logging categories.